styles/styles.css

```diff
@@ -0,0 +1,264 @@
+body {
+    font-family: 'Berkshire Swash', cursive;
+}
+p, nav {
+    font-family: 'Anaheim', sans-serif;
+}
+/*Header*/
+.header {
+    background-image: url(/images/promo-background.jpeg);
+    height: 420px;
+    width: 99.5%vw; 
+    color: white;
+}
+.top-container {
+    display: flex;
+}
+/*Logo*/
+.logo {
+    margin-top: 10px;
+    margin-left: 40px;
+    margin-right: -60px;
+}
+/*End of logo*/
+ul {
+    list-style-type: none;
+    display: inline-flex;
+    padding: 20px;
+    color: white;
+    font-family: 'Berkshire Swash', cursive;;
+    font-size: small;
+    margin-left: 240px;
+}
+ul a {
```

Comment:
Ah I see now. These `<ul>` elements are for the header. I would give them a class and not rely on the `<ul>` tag